jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsResourceCache.kt

```diff
@@ -390,4 +391,7 @@ class DefaultAwsResourceCache(
             return removed
         }
     }
+
+    override fun dispose() {
```

Comment:
Why do we need this? If we are going to dispose should we not clear out the cahce?

Reply:
We need this because the Disposable interface that we are using to register to the message bus has no default impl, but good call out, we should clear the cache hash map just in case here as well